modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -64,8 +64,8 @@ def fetch_errored(claim)
 
       def format_evss_errors(errors)
         errors.map do |error|
-          formatted = error['key'] ? error['key'].gsub('.', '/') : error['key']
-          { status: 422, detail: "#{error['severity']} #{error['detail'] || error['text']}".squish, source: formatted }
+          formatted = error[:key] ? error[:key].gsub('.', '/') : error[:key]
+          { status: 422, detail: "#{error[:severity]} #{error[:detail] || error[:text]}".squish, source: formatted }
         end
```

Comment:
@stiehlrod  Do we need to symbolize these keys as well to make sure any older error message get read too?